### ISSUE TITLE
shawl won't build due to outdated windows-service dependency.

### ISSUE THREAD

**Issue by TheGoddessInari** [mtkennerly/shawl#8]
The version of windows-service you use depends on a bad version of err-derive that uses a private symbol in `quote`.

It fails to build on both stable and nightly Rust.

```
145 | fn display_body(s: &synstructure::Structure) -> Option<quote::__rt::TokenStream> {
    |                                                               ^^^^ could not find `__rt` in `quote`

error: aborting due to previous error
```

This requires a new crates.io release for anyone to use shawl at the present time.

**Comment by mtkennerly** [comment]
Thanks for letting me know about this. I'll make a new release to address this soon.

**Comment by mtkennerly** [comment]
This is now fixed in v0.6.1.
